operators.js

```diff
@@ -396,9 +397,14 @@ function toPullStream() {
           (err) => cb(err)
         )
       }),
-      pull.map((ops) =>
-        cat([paginateStream(ops), meta.live ? meta.db.live(ops) : null])
-      ),
+      pull.map((ops) => {
+        if (meta.liveOnly) return meta.db.live(ops)
+        else
+          return cat([
+            paginateStream(ops),
+            meta.live ? meta.db.live(ops) : null,
+          ])
```

Comment:
This is totally fine implementation, but to nitpick a tiny bit, how about an `else if (meta.live) return cat([.....])` and then an `else return paginateStream(ops)`? Just to satisfy the eye :)

Reply:
Good idea